### ISSUE TITLE
Testing bundle_validator failed using optimized builds on CI.

### ISSUE THREAD

**Issue by oopsmonk** [iotaledger/entangled#347]
```
==================== Test output for //consensus/bundle_validator/tests:bundle_validator:
consensus/bundle_validator/tests/bundle_validator.c:50:test_bundle_validate_empty_invalid:FAIL: Expression Evaluated To FALSE
consensus/bundle_validator/tests/bundle_validator.c:60:test_bundle_size_1_value_with_wrong_address_invalid:FAIL: Expression Evaluated To FALSE
consensus/bundle_validator/tests/bundle_validator.c:142:test_bundle_exceed_supply_pos_invalid:FAIL: Expression Evaluated To FALSE
consensus/bundle_validator/tests/bundle_validator.c:187:test_bundle_exceed_supply_neg_invalid:FAIL: Expression Evaluated To FALSE
consensus/bundle_validator/tests/bundle_validator.c:232:test_bundle_validate_size_4_value_wrong_sig_invalid:FAIL: Expression Evaluated To FALSE
consensus/bundle_validator/tests/bundle_validator.c:302:test_bundle_validate_size_4_value_valid:FAIL: Expression Evaluated To FALSE
```
CI builds: 
https://buildkite.com/iota-foundation/entangled-prs/builds/761
https://buildkite.com/iota-foundation/entangled-prs/builds/770

**Comment by ifullgaz** [comment]
Fixed in PR#346. Not strictly related to flex_trits, but CI was failing, as noted by OP

**Comment by oopsmonk** [comment]
fixed in #350
